.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -141,5 +141,16 @@ body:
   - type: input
     id: device-model
     attributes:
-      label: Real device model
-      placeholder: ex. Samsung Galaxy A22
+      label: Device model
+      description: On what device you are experiencing this problem? Specify full device name and the name and version of the operating system it's running.
+      placeholder: ex. Samsung Galaxy A22 (Android 12)
+
+  - type: dropdown
+    id: acknowledgements
+    attributes:
+      label: Acknowledgements
+      description: I searched for similar issues in the repository.
+      options:
+        - 'Yes'
+    validations:
+      required: true
```

Comment:
Do we really need this?

Reply:
![triggered](https://user-images.githubusercontent.com/39658211/184082015-ffc1da09-8db0-46bc-8113-b3fd44d0b3a5.gif)